client/foundries.go

```diff
@@ -242,6 +242,23 @@ type TargetTestList struct {
 	Next  *string      `json:"next"`
 }
 
+type Wave struct {
+	Name       string           `json:"name"`
+	Version    string           `json:"version"`
+	Tag        string           `json:"tags"`
+	Targets    *json.RawMessage `json:"targets"`
+	CreatedAt  string           `json:"created-at"`
+	FinishedAt string           `json:"finished-at"`
+	Status     string           `json:"status"`
+}
+
+type WaveCreate struct {
+	Name    string     `json:"name"`
+	Version string     `json:"version"`
+	Tag     string     `json:"tag"`
+	Targets tuf.Signed `json:"targets"`
```

Comment:
So, we actually passing an overall target.json from a client to the backend, I remember talking about sending just signature?

Reply:
I've reconsidered this during a discussion with @doanac .
Yes, we send entire JSON as it allows to keep this logic of constructing a `targets.json` in a single place (client side) that's good from both consistency and performance points of view, and also allows to keep an implementation simple.